dart/examples/io_example/main.dart

```diff
@@ -25,7 +25,19 @@ Future<void> main(List<String> rawArgs) async {
   // Sends a full Sentry event payload to show the different parts of the UI.
   final sentryId = await Sentry.captureEvent(event);
 
-  print('SentryId : ${sentryId}');
+  print('Capture event result : SentryId : ${sentryId}');
```

Comment:
don't they need a `toString()`?

Reply:
the `toString()` is implicit when you interpolate an object in a `String`